libraries/compound/build.gradle.kts

```diff
@@ -12,13 +12,6 @@ plugins {
 android {
     namespace = "io.element.android.compound"
 
-    defaultConfig {
-        vectorDrawables {
-            useSupportLibrary = true
-            generatedDensities()
-        }
-    }
-
```

Comment:
I think not having this caused issues when trying to use vector icons in old API versions. It may be worth it to double check this: https://github.com/element-hq/compound-android/pull/1

Maybe now they're part of the same project and we bumped the min API version to 24 we won't have this issue.

Reply:
I checked on API 24 and there is no crash, but I will double check again.

Reply:
OK, so there is no crash on API 24, but we can see that some icons are broken. It's not new, so it's not blocking this PR.

<img width="1244" height="600" alt="image" src="https://github.com/user-attachments/assets/a630b8dc-0ad1-4192-8dab-180fbfd9e6ff" />